man/adoc/bpftrace.adoc

```diff
@@ -3793,6 +3793,15 @@ Available options:
 If the DebugInfo was rewritten by a post-linkage optimisation tool (like BOLT or AutoFDO), it might yield an incorrect address for a probe location.
 This config can force using the Symbol Table, for when the DebugInfo returns invalid addresses.
 
+**NOTE**: the following config variables are experimental (as indicated by their "unstable" prefix) so use these with caution and be aware they may be changed in future releases.
+
+=== unstable_attach_to_inline_funcs
```

Comment:
Might be good practice to link each unstable feature to a GH issue. And in the issue describe what work is left to do or what criteria we'll use to stabilize. Just so users have a way to gauge if it's usable for them. And also for us to remember all the in-flight work.

@ttreyer can you help file the issue?

Reply:
ping @ttreyer